style.css

```diff
@@ -14,7 +14,7 @@ body {
 .wrap{
     width: 1090px;
     border: solid 1px #dcdcdc;
-    height: 100%;
+    height: 5000px;
```

Comment:
Are you sure, that 5000px is a good idea?

Reply:
when I set a percent value, background also get lost